src/main.c

```diff
@@ -426,7 +426,11 @@ int main(int argc, char *argv[])
             break;
         }
 
-        usleep(10000);
```

Comment:
Without the sleep, the search thread will take much more CPU ...

Reply:
You're right! Especially for long and complex searches.